db/db_impl/db_impl_write.cc

```diff
@@ -80,14 +79,23 @@ void DBImpl::PebbleWriteCommit(CommitRequest* request) {
   }
 }
 
+Status DBImpl::MultiBatchWrite(const WriteOptions& options,
+                           std::vector<WriteBatch*>&& updates) {
+  if (immutable_db_options_.enable_pipelined_commit) {
```

Comment:
Actually, I believe name should be consistent. How about enable_multi_batch_write?

Reply:
And please add some description for the algorithm (and maybe credit to pebble) in the comments for this flag.

Reply:
Done, thanks!